developer-docs/how-to-guide/consuming_telemetry_data.md

```diff
@@ -0,0 +1,305 @@
+
+This API returns the dataset based on **fromDate** and **toDate**.
+
+Response body sample with indicative values:
+
+``` 
+{
+    "id": "org.xyz.analytics.telemetry",
+    "ver": "1.0",
+    "ts": "2018-11-15T12:35:37.522+00:00",
+    "params": {
+        "resmsgid": "e7e8f75d-fcc2-4f4f-96a1-ab48257591b5",
+        "status": "successful",
+        "client_key": null
+    },
+    "responseCode": "OK",
+    "result": {
+        "telemetryURLs": [
+            **"https://xyzproductionall.blob.core.windows.net/telemetry-data-store/channel/01235953112556064029750/raw/2018-10-01-1538393108525.json.gz"**
+        ],
+        "expiresAt": 1542287137444
+    }
+}
+```
+
+<b> Note </b>:
+
+fromDate and toDate must be in YYYY-MM-DD format.
+toDate must be greater than or equal to fromDate and toDate must be less than today. 
+
+* Maximum one month’s (31 days) data could be downloaded in one API call.
+
+* The API key is required for accessing the data exhaust. 
+
+<b> Response Code </b>:
+
+In case of error, an appropriate error response is returned.
+
+400 BAD REQUEST: Submit Data Request operation failed! The possible reason for failure is that you may have missed providing input for a mandatory parameter
+
+500 INTERNAL SERVER ERROR: The error can be identified by browsing through the API server logs or contact the server admin to retrieve the API logs.
+
+403 FORBIDDEN: If the API Key does not have access to the Channel Id specified in the request, the request will be forbidden from accessing the data exhaust.
+
+For e.g, the response body contains the following error message in case of FORBIDDEN error:
+Given X-Consumer-ID='9293548c-3a56-47dd-a9a3-24da3c821638' and X-Channel-ID='012222551180382207027' are not authorized
+
+<b>Extracting Data</b>:
+
+Using the telemetry URL that you recieve as part of the the response body of data exhaust API, you can download and extract the file ( .gzip format) that contain telemetry log.
+
+* Store the extracted json file to server/cloud storage for subsequent analysis.
+
+### Analyzing the Data
+
+Analyst can now retrive the data stored on the server/cloud storage and analyse the data based on the use case and scenario using some analytics tool as per your choice.
+
+1. Analyst may apply certain filters based on the use case for analysis of the data.
+For example, to compute app session or portal session time, you can apply filtes as mentioned below: 
+
+For App content sessions:
+
+"dimensions.pdata.id" = "prod.diksha.app" & "dimensions.mode" = "play" & "dimensions.type" in ("content","player")
+
+For Portal content sessions:
+
+"dimensions.pdata.id" = "prod.diksha.portal" & "dimensions.mode" = "play" & "dimensions.type" = "content"
+
+2. Another example: To calculate the overall time spent on a content, analyst should use time_spent parameter under the eks section in the edata field from the [workflow summary](https://github.com/ekstep/Common-Design/wiki/[Data-Product]-Workflow-Summarizer) data.

```

Comment:
@seventunes this URL is not accessible to the public user. It can be accessed by the user who has access to ekstep repo. So suggest you move this content somewhere else.
@Basreena @Sunitashelar